src/command_modules/azure-cli-reservations/azure/cli/command_modules/reservations/_help.py

```diff
@@ -0,0 +1,142 @@
+﻿# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.help_files import helps
+# pylint: disable=line-too-long
```

Comment:
You don't need this. None of the lines in this file is too long.

Reply:
deleted